tools/utils/typescript.js

```diff
@@ -0,0 +1,53 @@
+import {
+  getRandomEntry,
+  getRandomInt,
+  getRandomNoun,
+  getRandomNounCapitalized,
+  getRandomVerb,
+} from "./helpers";
+
+export default class TypeScript {
+  static getRandomFunctionName() {
+    return `${getRandomVerb()}${getRandomNounCapitalized()}`;
+  }
+
+  static getRandomVariableDeclaration() {
+    const keyWords = ["let", "const", "var"];
+    const options = ["[]", "this"];
```

Comment:
We have more options in the JS file now - perhaps a helper and use the same across JS and TS? ;)